### DISCUSSION TITLE
Installed 1.3.3.8 and installed the K1 scripts now Klipper won't start

### DISCUSSION THREAD

**Discussion by K2AAE** [Guilouz/Creality-Helper-Script-Wiki#468] [Issues]
{"code": "key341", "msg": "Section 'server' is not a valid config section\n\nOnce the underlying issue is corrected, use the \"RESTART\"\ncommand to reload the config and restart the host software.\nPrinter is halted\n", "values": ["server"]}

I've been thru every file in /usr/data/printer_data/config/ - found one Server section in moonraker.conf, commented it out still the same error.

I let Creality 'upgrade' to 1.3.3.8, had to use the complete reinstall method to get rid of the old crap, works now.

**Comment by Guilouz** [comment]
After updating firmware you must reset to factory settings.

**Reply by K2AAE** [reply]
When I said it was running, that was the new firmware after a complete reset, it worked until I ran the script, installed items: 1,2,4,5,14,6 and now I get:
{"code": "key341", "msg": "Section 'server' is not a valid config section\n\nOnce the underlying issue is corrected, use the \"RESTART\"\ncommand to reload the config and restart the host software.\nPrinter is halted\n", "values": ["server"]}

Again.

Ok, here is what caused that: After installing 1 ~ 6 + 14 I opened the browser and was told moonraker was not running, I added moonraker.conf to the printer.cfg as an include. I just removed it and the error above disappeared. So, Install firmware, hard reboot, install script, hard reboot. Hard reboot = power cycle

**Comment by TheWebMachine** [comment]
Sorry...that was supposed to post as a new discussion...not sure how it ended up here. 🤦🏻‍♂️

**Comment by GrandpaJ152** [comment]
i just did the root on a K1 Max and works great
